doc/Language/regexes.pod6

```diff
@@ -15,8 +15,8 @@ those patterns to actual text.
 Perl 6 has special syntax for writing regexes:
 
     m/abc/;         # a regex that is immediately matched against $_
-    rx/abc/;        # a Regex object
-    /abc/;          # a Regex object
+    rx/abc/;        # a Regex object; used to store regex object in a variable
+    /abc/;          # a Regex object; shorthand version of 'm/ /' operator
```

Comment:
As you write below, `/abc/` is different from `m/abc/`, it might replace the "shorthand" part.

`/abc/` is shorthand version of `rx/abc/` though, except `rx{abc}` is legal but `{abc}` means other thing.